src/app/layouts/PageWithImageBg.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, ReactNode } from 'react';
```

Comment:
It's better to move this component to templates/ folder

U will see similar components in this folder as well